fly_io/assets/logs/fly-io_tests.yaml

```
@@ -0,0 +1,88 @@
+id: "fly-io"
+tests:
+ -
+  sample: |-
+    {
+      "fly" : {
+        "app" : {
+          "instance" : "32882401c3294",
+          "name" : "datadog-agent-example"
+        },
+        "region" : "ewr"
+      },
+      "log" : {
+        "level" : "info"
+      },
+      "host" : "32882401c3294",
+      "env" : "flyenv",
+      "event" : {
+        "provider" : "app"
+      },
+      "message" : "2024-07-17 15:12:03 UTC | TRACE | INFO | (pkg/trace/info/stats.go:98 in LogAndResetStats) | [lang:python lang_version:3.8.19 interpreter:CPython tracer_version:2.8.5 endpoint_version:v0.5 service:fly-io-django-example-app] -> traces received: 2, traces filtered: 0, traces amount: 7118 bytes, events extracted: 0, events sampled: 0",
+      "ddsource" : "fly_io",
```

We should updated to `fly-io` as it's defined in `fly-io.yaml`
```suggestion
      "ddsource" : "fly-io",
```

---

Good catch! I want `fly_io` to be the source so I ended up making that the ID and then making `fly-io` the metric ID. Let me know if that looks good!